gz_ros2_control/gz_hardware_plugins.xml

```diff
@@ -0,0 +1,10 @@
+<library path="gz_hardware_plugins">
```

Comment:
Git is treating this as a new file. Can you use `git mv` on the `ign` version of this file so it's treated as a rename?